src/app/docs/button/button-docs.component.html

```diff
@@ -95,6 +95,29 @@
               </p>
             </td>
           </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <button
+                type="button"
+                class="sky-btn sky-btn-icon-borderless"
+              >
+              <i [skyThemeClass]="{
+                'fa fa-pencil': 'default',
+                'sky-icon sky-i-edit': 'modern'
+              }"></i>
+              </button>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Borderless icon buttons trigger common actions that are effectively represented using only icons, such as edit and delete. Only use them when there is not enough space to use a regular button with an icon, such as in a repeater row to invoke an inline form.
```

Comment:
```suggestion
                Icon buttons trigger common actions, such as edit and delete, when space is too tight for text labels and icons are sufficient to represent actions. For example, use icon buttons to invoke inline forms from repeater rows.
```
A few things:
- Just my 2 cents, but I don't think we need `Borderless` as an adjective. We don't use it in the descriptions for the link and inline link button classes, and beyond consistency, the adjective here suggests the possibility of an icon button WITH a border, which I think we should avoid implying is an option.
- The "such as" clause was a misplaced modifier, so I moved it.
- I reorganized things to get the angle about space closer to the beginning to make sure it doesn't get overlooked. Without that info front and center, I think someone might infer that they can use this button type anytime they have an icon that doesn't require a label for clarity.